### PR TITLE
FIX tests getting killed

### PR DESCRIPTION
**Pull request by fabianp**

### CODE REVIEW

File: requirements_test.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ optax>=0.0.9
 pytest-xdist
 scikit-learn>=0.24.1
 cvxopt
+pytest-xdist
```
Comment (mblondel) [inline requirements_test.txt:7]: See line 4 :)

Comment (fabianp) [reply requirements_test.txt:7]: LOL I'm dense

Comment (fabianp) [reply requirements_test.txt]: removed this line. Let's see if tests pass

**Review by mblondel — APPROVED** [review]
Thank you

**Pull request merged**